node/cn/tracers/api.go

```diff
@@ -109,23 +109,23 @@ func NewUnsafeAPI(backend Backend) *API {
 }
 
 type chainContext struct {
-	api *API
```

Comment:
Is this change necessary, or a minor improvement?

Reply:
Yes, It's a minor improvement for calling the `newChainContext` private function with no API instance (not receiver function).

Reply:
OK, thank you.